Info-Orbs/include/widgets/webDataWidget.h

```diff
@@ -1,28 +1,28 @@
 #ifndef WEB_DATA_WIDGET_H
 #define WEB_DATA_WIDGET_H
 
-#include <widget.h>
 #include <ArduinoJson.h>
 #include <HTTPClient.h>
 #include <widget.h>
-#include "webDataModel.h"
+
+#include "model/webDataModel.h"
 #include "utils.h"
 
 class WebDataWidget : public Widget {
-public:
+   public:
     WebDataWidget(ScreenManager &manager, String url);
     ~WebDataWidget() override;
     void setup() override;
     void update(bool force = false) override;
     void draw(bool force = false) override;
     void changeMode() override;
 
-private:
+   private:
```

Comment:
Is this indenting done by a `.clang-format` file or something like it?
I am not seeing one in the project, so it might be an idea to add one in the foreseeable future.
This helps contributors maintain the same style.

Reply:
ps: I think this indenting style is fairly weird, with the access specifier indented one space less than the rest.

Reply:
I know it was in the project config at one point but I am not seeing it anymore. This is what I have in my VS Code settings.json
`"C_Cpp.clang_format_fallbackStyle": "{ BasedOnStyle: Google, IndentWidth: 4, ColumnLimit: 0}",`
I agree it seems weird. I am just going with it because it seems to be a standard.